app/javascript/app/components/my-climate-watch/my-account/my-account-styles.scss

```diff
@@ -0,0 +1,62 @@
+@import '~styles/layout.scss';
+
+.title,
+.disclaimer {
+  font-weight: $font-weight;
+  color: $theme-color;
+}
+
+.title {
+  font-size: $font-size-large;
+  margin-bottom: $gutter-padding;
+}
+
+.disclaimer {
+  font-size: $font-size-sm;
+  padding-left: 10px;
+}
+
+.input,
+.dropdown,
+.textarea,
+.optIn,
+.updateButton {
+  margin-bottom: $gutter-padding;
+}
+
+.textarea {
+  height: 200px;
```

Comment:
Could we add some padding-top: 1em; or maybe a little bit less to keep that gap from the top of the input. Maybe we have to check in other browsers but in Chrome it feels a bit weird